distribution/dashboard/ui/src/services/tenant.ts

```diff
@@ -98,8 +104,65 @@ export async function DeletePolicyOfTenant({
   });
 }
 
+export async function getBackupPolicy({
+  ns,
+  name,
+}: API.NamespaceAndName): Promise<API.BackupPolicyResponse> {
+  let r = await request(`${tenantPrefix}/${ns}/${name}/backupPolicy`);
+  let res: API.BackupPolicy = {
+    destType: '',
+    archivePath: '',
+    bakDataPath: '',
+    scheduleType: '',
+    scheduleTime: '',
+    scheduleDates: [],
+  };
+  if (r.successful) {
+    Object.keys(res).forEach((key) => {
+      res[key] = r.data[key];
+    });
+    return {
+      ...r,
+      data: res,
+    };
+  }
+  return r;
+}
+
+export async function getBackupJobs({
+  ns,
+  name,
+  type,
+  limit,
```

Comment:
limit = 10
Directly assigning default values to formal parameters would be more intuitive

Reply:
get